src/device-registry/controllers/create-activity.js

```diff
@@ -1,17 +1,17 @@
 const HTTPStatus = require("http-status");
 const isEmpty = require("is-empty");
-const { logObject, logElement, logText } = require("../utils/log");
+const { logObject, logElement, logText } = require("@utils/log");
```

Comment:
Like in the entire microservice, just removed length path names using a new npm package called [module-alias](https://www.npmjs.com/package/module-alias). Pretty good one to be honest. :)  @BenjaminSsempala